### PR TITLE
Start of compound text module.

### PR DESCRIPTION
**Pull request by angaither**
@aaronschachter ehhh

### CODE REVIEW

File: modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+   return array(
+    'columns' => array(
+      'ds_title' => array(
+        'type' => 'varchar',
+        'length' => 255,
+        'not null' => FALSE,
+      ),
+      'ds_copy' => array(
+        'type' => 'varchar',
+        'length' => 99999,
```
Comment (mchitten) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install]: I think when you reach a varchar of length 99,999 you'd be safer to go with some sort of text field.

Comment (mchitten) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install]: Addendum to this: I'm not actually sure that it's possible to have a varchar of length 99,999.  It seems that the [max length is 65,536](http://www.pythian.com/blog/text-vs-varchar/#more-7129)...which is still kind of ridiculous.

Comment (desmondmorris) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install]: +1 for text type

File: modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ */
+
+
+/**
+ * Implements hook_field_schema().
+ */
+function dosomething_compoundtext_field_schema($field) {
+   return array(
+    'columns' => array(
+      'ds_title' => array(
```
Comment (aaronschachter) [inline modules/dosomething/dosomething_compoundtext/dosomething_compoundtext.install]: I think the code will end up being more legible if calling these columns text1 and text2, or value1 and value2, vs having the ds_ prefix.

**Pull request closed without merging**